pyomo/util/tests/test_latex_printer.py

```diff
@@ -0,0 +1,349 @@
+#  ___________________________________________________________________________
+#
+#  Pyomo: Python Optimization Modeling Objects
+#  Copyright (c) 2008-2023
+#  National Technology and Engineering Solutions of Sandia, LLC
+#  Under the terms of Contract DE-NA0003525 with National Technology and
+#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain
+#  rights in this software.
+#  This software is distributed under the 3-clause BSD License.
+#  ___________________________________________________________________________
+
+import pyomo.common.unittest as unittest
+from pyomo.util.latex_printer import latex_printer
+import pyomo.environ as pe
+
+
+def generate_model():
+    import pyomo.environ as pe
```

Comment:
No need to re-import

Reply:
I prefer to have the generate model function be copy-paste-able and believe this import is of minimal overhead